app/script/conversation/ConversationRepository.coffee

```diff
@@ -101,6 +105,44 @@ class z.conversation.ConversationRepository
     amplify.subscribe z.event.WebApp.SELF.CLIENT_ADD, @on_self_client_add
     amplify.subscribe z.event.WebApp.USER.UNBLOCKED, @unblocked_user
 
+  ###
+  Adds a generic message to a the sending queue.
+
+  @private
+  @param conversation_id [String] Conversation ID
+  @param generic_message [z.protobuf.GenericMessage] Protobuf message to be encrypted and send
+  @param user_ids [Array<String>] Optional array of user IDs to limit sending to
+  @return [Promise] Promise that resolves when the message was sent
+  ###
+  _add_to_sending_queue: (conversation_id, generic_message, user_ids) =>
+    return new Promise (resolve, reject) =>
+      queue_entry =
+        function: => @_send_generic_message conversation_id, generic_message, user_ids
+        resolve: resolve
+        reject: reject
+
+      @sending_queue.push queue_entry
+
+  ###
+  Sends a generic message from the sending queue.
+  @private
+  ###
+  _execute_from_sending_queue: (queued_promises) =>
+    return if @block_event_handling or @sending_blocked
+    return if not queued_promises.length
+
+    queue_entry = @sending_queue()[0]
+    @sending_blocked = true
+    queue_entry.function()
+    .catch (error) =>
+      console.log 1
```

Comment:
What's even worse is that it's printing a number instead of a random (or more specific) text :)